### PR TITLE
adding a nuke script for easy cleanup

### PR DESCRIPTION
**Pull request by julien51**
Will first check if servers are still running.

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: scripts/nuke.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Then check if next is running
+NEXT_PORT=3000
+lsof -i :$NEXT_PORT >> /dev/null
+if [ $? -eq 0 ]; then
+  echo "Please make sure you stop the nextjs application first"
+  exit 1
+fi
+
+# Then check if locksmith is running
+LOCKSMITH_PORT=3000
```
Comment (julien51) [inline scripts/nuke.sh]:
```suggestion
LOCKSMITH_PORT=8080
```

**Comment by julien51** [comment]
Fixed it! Thanks for the careful look!

**Comment by akeem** [comment]
👍

File: scripts/nuke.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# To only be used after you've stopped all servers
+
+NUKE_DIRS="node_modules unlock-app/node_modules/ locksmith/node_modules/ tests/node_modules/ smart-contracts/node_modules/ unlock-app/build unlock-app/coverage unlock-app/src/out unlock-app/src/.next unlock-app/coverage unlock-app/src/out unlock-app/src/.next smart-contracts/build/contracts/*.json"
+
+# First check if ganache is running
+GANACHE_PORT=8545
+lsof -i :$GANACHE_PORT >> /dev/null
+if [ $? -eq 0 ]; then
+  echo "Please make sure you stop ganache first"
+  exit 1
```
Comment (nfurfaro) [inline scripts/nuke.sh]: What's the best way to do this? Sometimes I seem to get an "orphaned" ganache process running in the background, even when it seems I've stopped ganache. I've been using `alias kill8545="lsof -t -i tcp:8545 | xargs kill"` in my `.bashrc`

Comment (julien51) [reply scripts/nuke.sh]: That should work. I did not want to normalize because this may be very much OS specific.

**Pull request merged**